.gitignore

```diff
@@ -34,3 +35,19 @@ Makefile*
 *.qmlproject.user.*
 
 build
+
+# KDE show hidden folder marker
+.directory
+
+
+# built tests executables
```

Comment:
Why executables are located into source dir?
Anyway a wildcard like `test/tst_*/tst_*` may be good idea

Reply:
> Why executables are located into source dir?

If you build the Nut with shadow build disabled on Linux the executables ends up there. 

The problem with the `test/tst_*/tst_*` pattern is the fact that there are matching source/header files in the repo like (test/tst_upgrades/tst_upgrades.cpp)